packages/discord.js/typings/index.d.ts

```diff
@@ -2453,10 +2453,27 @@ export class Webhook extends WebhookMixin() {
   public sourceChannel: NewsChannel | APIPartialChannel | null;
   public token: string | null;
   public type: WebhookType;
-  public isIncoming(): this is this & { token: string };
+  public applicationId: Snowflake | null;
+  public isUserCreated(): this is this & {
+    type: WebhookType.Incoming;
+    applicationId: null;
+    owner: User | APIUser;
+    token: null;
```

Comment:
I wouldn't document this as `null`. Simply fetching a webhook in a channel will return the token.

Reply:
So `string | null` I assume?

Reply:
Hmm. The issue here is that if it's fetched with the owner, the token is not sent back, and I assume it's the same the other way round.